src/app/ViewRoadmap.tsx

```diff
@@ -61,7 +72,14 @@ export const ViewRoadmap: React.FC<ViewRoadmapProps> = ({ className = "" }) => {
                 <div className="roadmapContainer">
                     <div id="roadmap" ref={roadmapRef}>
                         {ROADMAP_ITEMS.map((item) => (
-                            <RoadmapItemComponent key={item.id} item={item} />
+                            <div
```

Comment:
Not a fan of wrapping each card in a new div just for that. Why not reusing the mechanisms for the 2 "left" "right" buttons, but make their hitbox larger instead?

Reply:
> should we compare the position of cards getBoundingClientRect() and their boundaries to decide scroll direction and reusing the scrollLeft , scrollRight

Sounds like a solution, I think it will integrate better with the existing code :+1: